src/benchmarkRunner/BenchmarkRunner.ts

```diff
@@ -59,26 +69,51 @@ class BenchmarkRunner extends BenchmarkCollection {
 			}
 		}
 
+		if (isLogExcellable) {
+			console.log('');
+		}
+
 		console.log(`Running ${comparisons.length} comparisons`);
+		if (isLogExcellable && comparisons.length) {
+			console.log(
+				'comparison name, test name, op/sec, comparison to first sample, rme, number of running'
+			);
+		}
+
 		for (const comparison of comparisons) {
-			console.log(`------------${comparison.name}------------`);
+			if (!isLogExcellable) {
+				console.log(`------------${comparison.name}------------`);
+			}
 
 			const operationsPerSecond: { name: string; ops: number }[] = [];
+			let myBase;
 			for (const benchmark of comparison.benchmarks) {
 				if (comparison.setup !== undefined) {
 					await comparison.setup();
 				}
 
 				benchmark.on('complete', (event: Event) => {
-					console.log(String(event.target));
+					if (isLogExcellable) {
+						const information = event.target as any;
+						if (!myBase) {
+							myBase = information.hz;
+						}
+						console.log(
+							`${comparison.name}, ${information.name}, ${information.hz}, ${
+								(information.hz / myBase) * 100
```

Comment:
I don't know if you have any preference, but could we do the comparison as the last argument?
To me it feels as one of the most important numbers and putting it as last should make it quicker to scan over the results and see what is faster.